packages/manager/apps/cloud/client/app/dbaas/logs/detail/aliases/home/logs-aliases-home.html

```diff
@@ -3,27 +3,19 @@
         text="::'logs_aliases_title' | translate"
     ></cui-page-content-title>
     <p data-translate="logs_aliases_description"></p>
-
-    <p>
-        {{ctrl.quota.data.configured}}/{{ctrl.quota.data.max}}
-        <span data-translate="logs_aliases_total_used"></span>
-        (<logs-upgrade-quota-link></logs-upgrade-quota-link>)
-    </p>
-
     <oui-datagrid rows="ctrl.aliases.data">
         <extra-top>
-            <logs-add-tool
-                data-ng-if="!ctrl.quota.loading"
-                text="{{ ::'logs_aliases_add' | translate }}"
-                tool-type="{{ ::'logs_aliases_name' | translate }}"
-                current-usage="ctrl.quota.data.configured"
-                max-usage="ctrl.quota.data.max"
-                callback="ctrl.add()"
+            <button
+                type="button"
+                class="oui-button oui-button_secondary"
+                data-ng-click="ctrl.add()"
             >
-            </logs-add-tool>
+                <i class="oui-icon oui-icon-add pr-1" aria-hidden="true"></i> {{
+                ::'logs_aliases_add' | translate }}
```

Comment:
```suggestion
                <span class="oui-icon oui-icon-add pr-1" aria-hidden="true"></span>
                <span data-translate="logs_aliases_add"></span>
```

Reply:
Ok, i also apply this on all others sections